test/zendesksupport/write-delete/main.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/test/utils"
+	"github.com/amp-labs/connectors/test/utils/mockutils"
+	connTest "github.com/amp-labs/connectors/test/zendesksupport"
+	"github.com/amp-labs/connectors/zendesksupport"
+)
+
+type brandsPayload struct {
+	Brand brand `json:"brand"`
+}
+type brand struct {
+	Subdomain string `json:"subdomain"`
+	Email     string `json:"email"`
+	Name      string `json:"name"`
+}
+
+var (
+	objectName = "brands" // nolint: gochecknoglobals
+)
+
+func main() {
+	// Handle Ctrl-C gracefully.
+	ctx, done := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	defer done()
+
+	// Set up slog logging.
+	utils.SetupLogging()
+
+	filePath := os.Getenv("ZENDESK_SUPPORT_CRED_FILE")
+	if filePath == "" {
+		filePath = "./zendesk-support-creds.json"
+	}
+
+	conn := connTest.GetZendeskSupportConnector(ctx, filePath)
+	defer utils.Close(conn)
+
+	fmt.Println("> TEST Create/Update/Delete brands")
+	fmt.Println("Creating brands")
+
+	view := createBrands(ctx, conn, &brandsPayload{
+		Brand: brand{
+			Subdomain: "nkn2",
```

Comment:
Instead of hard coding test values, can you use `gofakeit`? Examples: https://github.com/amp-labs/connectors/blob/main/test/hubspot/write/main.go

Reply:
Running the script about 5 times produced the successful outcome.
Subdomain cannot repeat and gofakeit is producing domains unique enough.
![image](https://github.com/user-attachments/assets/24312895-af62-41f8-a567-c77851b927ad)